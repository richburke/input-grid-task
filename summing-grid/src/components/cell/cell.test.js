import React from 'react';
import { shallow } from 'enzyme';
import Cell from './';

describe('Cell', () => {
  it('should render correctly', () => {
    const component = shallow(<Cell />);
    expect(component).toMatchSnapshot();
  });

  it('should render its container <div>', () => {
    const component = shallow(<Cell />);
    expect(component.find('div.grid-cell')).toBeDefined();
  });
  
  it('should contain the child passed to it', () => {
    const component = shallow(
      <Cell>
        <div key='cell-contents' />
      </Cell>
    );
    expect(component.contains(<div key='cell-contents' />)).toBeTruthy();
  });
});
