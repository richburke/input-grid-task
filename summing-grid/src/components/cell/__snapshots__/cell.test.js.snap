// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell should render correctly 1`] = `
<div
  className="grid-cell"
/>
`;
